Source/WebKit/UIProcess/API/Cocoa/_WKWebAuthenticationPanel.mm

```diff
@@ -1211,6 +1211,19 @@ + (NSData *)encodeMakeCredentialCommandWithClientDataHash:(NSData *)clientDataHa
     return encodedCommand.autorelease();
 }
 
++ (NSData *) encodeMakeCredentialCommandWithClientDataHash:(NSData *)clientDataHash options:(_WKPublicKeyCredentialCreationOptions *)options userVerificationAvailability:(_WKWebAuthenticationUserVerificationAvailability)userVerificationAvailability authenticatorSupportedCredentialParameters:(NSArray<_WKPublicKeyCredentialParameters *> *)credentialParameters
```

Comment:
This appears to be solely for testing, is that right?

Reply:
Yes, we don't have another way to check the generated CTAP request bytes in our testing framework right now.